src/topology.md

```diff
@@ -19,6 +25,14 @@ There are three kinds of route that a node may need to generate:
 - From a mixnode to the node. For SURBs (sent in requests to enable replies).
 - From the node to itself. For loop cover traffic.
 
+// TODO here make it explicit if "non-mixnode" are "node".
+// if so also it implies there is no mixnode to mixnode or cover between mixnode (this should maybe
```

Comment:
Not sure why this is implied? Mixnodes should generate cover traffic in pretty much the same way as non-mixnodes, they just don't need to worry about the gateway stuff.

Reply:
as in a previous reply, I really think that non mixnode cannot generate cover traffic as mixnode (exept if their number is bounded, but then we need to define how their priority queue works).